src/wallet/rpcdump.cpp

```diff
@@ -572,7 +576,9 @@ UniValue dumpwallet(const JSONRPCRequest& request)
     EnsureWalletIsUnlocked();
 
     ofstream file;
-    file.open(request.params[0].get_str().c_str());
+    boost::filesystem::path filepath = request.params[0].get_str();
+    filepath = absolute(filepath);
```

Comment:
Thanx.
IMO, it is better to inform users that path argument is from the viewpoint of bitcoind either absolute nor relative.
The problem is that users expects path argument is the viewpoint of RPC-caller, but actually the one of bitcoind. The former is not same as the latter even if absolute path if they executed on different docker.
So i added doc and output at this time.